examples/simple/igraph_assortativity.c

```diff
@@ -0,0 +1,32 @@
+/* -*- mode: C -*-  */
```

Comment:
Please remove this. Remember that the example is included in the documentation, and is aimed at all readers, including those who don't use emacs.

Reply:
Sorry, I didn't think of that. I'll remove it.